tests/python/test_function_parameter_by_value.py

```diff
@@ -15,3 +15,21 @@ def task():
 
     task()
     assert ret[None] == 0
+
+
+@ti.all_archs
+def test_kernel_argument_pass_by_value():
+    @ti.kernel
+    def task(x1: ti.i32, x2: ti.i32):
+        x1 = x1
```

Comment:
I'm not exactly sure what the problem is. From the title it sounds like we cannot assign kernel arg `x1` to other vars in the kernel? If so, could you assign them to another name, e.g. `x3 = x1`? (Not sure if this suggestion makes sense, though) Or is this self assignment (rare to see) intentional?